packages/backend/src/apps/lettersg/actions/create-letter/index.ts

```diff
@@ -101,15 +146,17 @@ const action: IRawAction = {
           $.app.name,
         )
       }
-      // TODO (mal): check which fields are not used and return
-      const lettersErrorData: LettersApiErrorData = error.response.data
-      if (lettersErrorData.message === 'Malformed bulk create object') {
-        throw new StepError(
-          'Missing pair of field/value for letter template',
-          'Click on set up action and check that you have entered all the fields and values in the letter parameters.',
-          $.step.position,
-          $.app.name,
-        )
+      // TODO (mal): check specific fields to return
+      if (error instanceof HttpError && error.response.status === 400) {
+        const lettersErrorData: LettersApiFieldErrorData = error.response.data
+        if (lettersErrorData.message === 'Invalid letter params.') {
```

Comment:
```suggestion
        if (lettersErrorData?.message === 'Invalid letter params.') {
```

Since the API is subject to change, and error messages are not exactly mission critical, we should be more lax with our checks. Add a `?` to prevent this from erroring out if letters stops returning this data.